packages/google-tag-manager/README.md

```diff
@@ -1,13 +1,67 @@
-> ## Beta version
->
-> Please, bear in mind that this package is currently in beta version. It hasn't proper tests or docs and its implementation may change with the final release.
+# `@frontity/google-tag-manager`
 
-# Google Tag Manager Analytics
+[![Version](https://img.shields.io/npm/v/@frontity/google-tag-manager.svg)](https://www.npmjs.com/package/@frontity/google-tag-manager) [![npm](https://img.shields.io/npm/dw/@frontity/google-tag-manager)](https://www.npmjs.com/package/@frontity/google-tag-manager) [![License: Apache--2.0](https://img.shields.io/badge/license-Apache%202-lightgrey)](https://github.com/frontity/frontity/blob/master/LICENSE)
 
-This package is used to add Google Tag Manager to your Frontity project.
+Analytics package to use Google Tag Manager with Frontity
+
+
+## Install
+
+```sh
+npm i @frontity/google-tag-manager
+```
 
 ## Usage
 
-You can install it with npm:
+Once is installed the Tracking ID should be added in the `frontity.settings.js` under the package `state`
+
+**`frontity.settings.js`**
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-tag-manager",
+      state: {
+        googleTagManager: {
+          containerIds: ['GTM-BCDFGHJ']
```

Comment:
For the basic usage I would show the singular option `containerId`:

- https://github.com/frontity/frontity/blob/dev/packages/google-tag-manager/types.ts#L19

The plural `containerIds` is only needed for advanced sites that need more than one container ID.

Reply:
Done! 👍